doc/changes/changes_0.7.0.md

```diff
@@ -0,0 +1,11 @@
+# Exasol Python Test Framework 0.7.0, released t.b.d.
+
+Code name: t.b.d.
+
+## Summary
+
+t.b.d.
+
+## Security
+
+ - #76: Updated dependencies
```

Comment:
To fix which CVEs? Please list.

Reply:
it is only a poetry update, not CVE needed fixing, because to the users of this package was never an issue visible